tests/jobs/rft/test_gendata_rft.py

```diff
@@ -0,0 +1,168 @@
+import pytest
+import tempfile
+import filecmp
+import shutil
+import os
+from distutils.dir_util import copy_tree
+
+from semeio.jobs.scripts.gendata_rft import main_entry_point
+
+ECL_BASE = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "data/norne/NORNE_ATW2013"
+)
+EXPECTED_RESULTS_PATH = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "data/norne/expected_result"
+)
+
+
+@pytest.fixture
+def input_data(tmpdir):
+    data_dir = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), "data/norne/gendata_rft_input_files"
+    )
+    copy_tree(data_dir, tmpdir.strpath)
+
+    cwd = os.getcwd()
+    tmpdir.chdir()
+
+    try:
+        yield
+
+    finally:
+        os.chdir(cwd)
+
+
+def test_gendata_rft_entry_point(tmpdir, input_data):
+
+    arguments = [
+        "-e",
+        ECL_BASE,
+        "-w",
+        "well_and_time.txt",
+        "-t",
+        tmpdir.strpath,
+        "-z",
+        "zonemap.txt",
+    ]
+
+    main_entry_point(arguments)
+
+    for expected_file in os.listdir(EXPECTED_RESULTS_PATH):
+        assert expected_file in os.listdir(tmpdir.strpath)
+        if not expected_file.endswith("inactive_info"):
+            assert filecmp.cmp(
+                expected_file, os.path.join(tmpdir.strpath, expected_file)
```

Comment:
So this is super stupid, but the `expected_file` is only the base name, and as such what this function does is to only check that the same file, the one in `tmpdir.strpath` is identical.. 

In fact, there was a bug in the `validate_zonemap` which caused every well that did not have a zone described to be given `-1` for pressure.